<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>LatitudeLongitudePanel</class>
  <widget class="QDialog" name="LatitudeLongitudePanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>306</width>
        <height>151</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>200</width>
        <height>100</height>
      </size>
    </property>
    <property name="baseSize">
      <size>
        <width>400</width>
        <height>200</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Latitude/Longitude</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>false</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>4</number>
      </property>
      <item>
        <layout class="QGridLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>4</number>
          </property>
          <item row="0" column="0">
            <widget class="QLabel" name="latitudeLabel">
              <property name="text">
                <string>Latitude:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="longitudeLabel">
              <property name="text">
                <string>Longitude:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLineEdit" name="latitudeLine">
              <property name="readOnly">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QLineEdit" name="longitudeLine">
              <property name="readOnly">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="QLineEdit" name="distanceLine">
              <property name="readOnly">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QCheckBox" name="distanceCheckBox">
              <property name="text">
                <string>Distance:</string>
              </property>
            </widget>
          </item>
          <item row="3" column="2">
            <widget class="QPushButton" name="okButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>OK</string>
              </property>
              <property name="shortcut">
                <string/>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="3" column="3">
            <widget class="QPushButton" name="cancelButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="shortcut">
                <string/>
              </property>
            </widget>
          </item>
          <item row="2" column="2">
            <widget class="QPushButton" name="defaultButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Default</string>
              </property>
            </widget>
          </item>
          <item row="2" column="3">
            <widget class="QPushButton" name="currentButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Current</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="4" margin="11"/>
  <pixmapfunction></pixmapfunction>
  <tabstops>
    <tabstop>latitudeLine</tabstop>
    <tabstop>longitudeLine</tabstop>
    <tabstop>distanceCheckBox</tabstop>
    <tabstop>distanceLine</tabstop>
    <tabstop>defaultButton</tabstop>
    <tabstop>currentButton</tabstop>
    <tabstop>okButton</tabstop>
    <tabstop>cancelButton</tabstop>
  </tabstops>
</ui>
